windows/src/desktop/kmshell/xml/strings.xml

```diff
@@ -1186,7 +1186,10 @@ keyboard that you use in Windows.  Keyman keyboards will adapt automatically to
   <!-- Introduced: 8.0.294.0 -->
   <string name="S_OSK_FontHelper_ChooseKeyboard" comment="OSK Font helper choose keyboard">Please select a Keyman keyboard to find related fonts.</string>
 
-
+  <!-- Context: OSK Font Helper -->
+  <!-- String Type: FormatString -->
+  <!-- Introduced: 17.0.312.0 -->
+  <string name="S_OSK_FontHelper_NoFonts" comment="OSK Font helper no fonts found for keyboard">No fonts have been found as suggestions for Keyboard %1$s.</string>

```

Comment:
@rc-swag a translator asked if Keyboard was intended to be capitalized here?

Reply:
> @rc-swag a translator asked if Keyboard was intended to be capitalized here?

Good catch by them, no it wasn't. it should be `keyboard` lower case.